dbt/models/default/default.pinval_assessment_card.sql

```diff
@@ -0,0 +1,285 @@
+{{
+    config(
+        materialized='table'
+    )
+}}
+
+WITH run_ids_to_include AS (
+    SELECT run_id
+    FROM {{ source('model', 'metadata') }}
+    -- This will eventually grab all run_ids where
+    -- run_type == comps
+    WHERE run_id = '2025-02-11-charming-eric'
+),
+
+school_data AS (
+    SELECT
+        pin10 AS school_pin,
+        year,
+        school_elementary_district_name,
+        school_secondary_district_name
+    FROM location.school
```

Comment:
Still having trouble trying to use `source()` to call this. Do I understand correctly that if the model is materialized as a table, the `{{ source('', '') }}` call should work?

Reply:
This is very clear, thank you!